plugins/social_network/social_network.cpp

```diff
@@ -221,7 +221,9 @@ namespace golos { namespace plugins { namespace social_network {
         if (itr != by_permlink_idx.end()) {
             return get_discussion(*itr, limit);
         }
-        return discussion();
+        auto res = discussion();
```

Comment:
Move the initialization of an empty discussion to the discussion_helper.
It is more useful, and doesn't require to call the follow::fill_account_reputation() directly from a random place.

Reply:
Couldn't include plugin.hpp (follow) from api (discussion_helper).